bson/src/main/org/bson/BasicBSONDecoder.java

```diff
@@ -22,11 +22,47 @@
 import java.io.InputStream;
 import java.nio.ByteBuffer;
 
+import static org.bson.assertions.Assertions.notNull;
+
 /**
  * Basic implementation of BSONDecoder interface that creates BasicBSONObject instances
  */
 public class BasicBSONDecoder implements BSONDecoder {
 
+    /**
+     * Sets the default {@link UuidRepresentation} to use when decoding BSON binary values.
+     *
+     * <p>
+     * If unset, the default is {@link UuidRepresentation#JAVA_LEGACY}.
```

Comment:
Nit: Should this just be: "Defaults to {@link UuidRepresentation#JAVA_LEGACY}."

Due to the notNull I cant see how its unset?